src/lib/common/include/sol-certificate.h

```diff
@@ -74,6 +77,8 @@ void sol_cert_unref(struct sol_cert *cert);
  */
 const char *sol_cert_get_filename(const struct sol_cert *cert);
 
+#endif /*SOL_FEATURE_FILESYSTEM*/
```

Comment:
On 12-05-2016 13:31, Otávio wrote:
> In src/lib/common/include/sol-certificate.h
> <https://github.com/solettaproject/soletta/pull/2025#discussion_r63052225>:
>
>> @@ -74,6 +77,8 @@ void sol_cert_unref(struct sol_cert *cert);
>>   */
>>  const char *sol_cert_get_filename(const struct sol_cert *cert);
>>
>> +#endif /*SOL_FEATURE_FILESYSTEM*/
>
> We only have a implementation for linux so far, that is using filesystem
> for all methods.

Fair then =D

>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/2025/files/ec6a29ab6849e8381656037cb22b22c07c938e77..5f905c3aaca5a0deab7400b8a8a25b565a948fb5#r63052225>
>